tests/testthat/indention_curly_brackets/one_line_curly-out.R

```diff
@@ -0,0 +1 @@
+a <- {1+1}
```

Comment:
this test is testing indention based on curly brackets *only* (with the transformer function `style_indent_curly()`, see tests/testthat/test-indention_curly.R#3+). Such tests are aimed to only test a subset of the styling. Over-all tests are done with transformer function `style_text()` and friends, and they would add the proper spaces for this example (tested elsewhere).